thirdeye/thirdeye-frontend/app/pods/components/contribution-table/template.hbs

```diff
@@ -0,0 +1,165 @@
+<div class="table-responsive analysis-table-wrapper ">
```

Comment:
Copied from old Thirdeye UI. Will need to revisit this to remove some template level logic